addons/binding/org.openhab.binding.logreader/README.md

```diff
@@ -88,9 +92,8 @@ rule "LogReader"
         // do something
     end
 ```
```

Comment:
add an empty line behind the code section

Reply:
Done